tutorials/3d/procedural_geometry/index.rst

```diff
@@ -126,14 +126,14 @@ Which method you use depends on what you are trying to do and what kind of proce
 
 Both SurfaceTool and ArrayMesh are excellent for generating static geometry (meshes) that don't change over time.
 
-Using an ArrayMesh is slightly faster than using a SurfaceTool, but the API is more a little more challenging.
+Using an ArrayMesh is slightly faster than using a SurfaceTool, but the API is a little more challenging.
 Additionally, SurfaceTool has a few quality of life methods such as ``generate_normals()`` and ``index()``.
 
 ImmediateGeometry regenerates the mesh every frame, so it is much slower than ArrayMesh or SurfaceTool. However, if you
-need the geometry to change every frame anyway it provides a much easier interface that may even be a little faster than generating
+need the geometry to change every frame anyways, it provides a much easier interface that may even be a little faster than generating
```

Comment:
"anyway" is technically correct here. "Anyways" is a slang word that is synonymous with "anyway". 

I'm not sure what usage is like outside of north America, but it is probably best to leave out the "s".

Reply:
Thanks for the input. I changed "anyways" to "anyway".